setup_yarn.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -e
+
+require_var() {
+  if [ -z "$1" ]; then
+    >&2 echo "$2"
+    return 1
+  fi
+  return 0
+}
+
+require_env() {
+  require_var "$2" "Missing env var '$1'"
+}
+
+setup_yarn() {
+
```

Comment:
Curious why there's an empty line here but none for all the other functions.

Reply:
because i'm blind as a bat :D

Reply:
"bat" -- pun intended? ;)